web/themes/custom/pineapple/sass/components/_hero-image.scss

```diff
@@ -22,36 +22,38 @@
     align-items: flex-end;
     margin-bottom: 100px;
 
-    .page-title {
-      padding: 15px;
-      background-color: transparentize($primary, 0.4);
-      font-size: $xlarge-font;
-      font-weight: 600;
-      position: absolute;
-      text-align: left;
-      color: $white;
-      max-width: 90%;
-
-      h1 {
-        margin: 0;
-      }
-    }
+    .hero-image-content {
+
+      .page-title {
+        padding: 15px;
```

Comment:
Create `$unit` variable as previously discussed and begin to use it for spacing consistency on the site

Reply:
done